src/main/java/com/github/dockerjava/api/command/AsyncDockerCmd.java

```diff
@@ -13,6 +13,6 @@
  */
 public interface AsyncDockerCmd<CMD_T extends AsyncDockerCmd<CMD_T, A_RES_T>, A_RES_T> extends DockerCmd<Void> {
 
-    public <T extends ResultCallback<A_RES_T>> T exec(T resultCallback);
+    <T extends ResultCallback<A_RES_T>> T exec(T resultCallback);
```

Comment:
@marcuslinke there are >500 "RedundantModifier" errors. And my IDEA also says the same.
Are you ok to clean them up?